README.md

```diff
@@ -31,3 +31,6 @@
 ## Information About Go
 
  1. Go Is Not OOP Nor POP Language
+
+ 2. Go (also called Golan or Go Language)is an open source programming language used for general purpose
```

Comment:
Just some corrections - It's GoLang not Golan

Also in the markdown ordered list, you don't have to insert a new line after one bullet point i.e. remove the new line from line no 34